### PR TITLE
[DEV-5242] More Precise Lines for Award Amounts Viz

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**
The award amounts viz works but the lines drawn to outline the bars in the description do not consider the padding or nested position of the bars.

**Technical details:**
For each award type and each spending category, I have a value for the approximate amount of padding. I apply that padding to the line via a margin and the width of the line.

I have an object that stores these values in a dataMapping file.

Admittedly, the exact padding does not always look right so I have done some hacking here to just make it look right.

Here's a contrast for comparison:

QAT:
image
LOCAL:
image

**JIRA Ticket:**
[DEV-5242](https://federal-spending-transparency.atlassian.net/browse/DEV-5242)

**Mockup:**
https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
`N/A` Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
`N/A` Verified mobile/tablet/desktop/monitor responsiveness
`N/A` Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
`N/A` Added Unit Tests for methods in Container Components, reducers, helper functions, and models `if applicable`
`N/A` All `componentWillReceiveProps` and `componentWillMount` in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3277](https://federal-spending-transparency.atlassian.net/browse/DEV-3277)
`N/A` [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
`N/A` [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [x] Design review complete `if applicable`
`N/A` [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: src/_scss/pages/award/shared/awardAmountsSection/_charts.scss [hunk 1]
```diff
@@ -1,6 +1,14 @@
 $exceeds-current-color: #fdb81e;
 $exceeds-potential-color: #e31c3d;
 
+// Offsets per DEV-5242:
+// refer to datamapping object; divide relevant value by 2.
+// for proper margin-left value of text offset.
+$proc-obligated-offset: rem(5);
```
Comment (ebdabbs) [inline src/_scss/pages/award/shared/awardAmountsSection/_charts.scss:7]: What does `proc` stand for here?

File: src/_scss/pages/award/shared/awardAmountsSection/_charts.scss [hunk 2]
```diff
@@ -189,6 +206,34 @@ $exceeds-potential-color: #e31c3d;
                 br {
                     font-size: $small-font-size;
                 }
+
+                // Offsets for Bar Value Descriptions; for example $10,000
```
Comment (ebdabbs) [inline src/_scss/pages/award/shared/awardAmountsSection/_charts.scss:210]: What is $10,000 and example of?

File: src/js/helpers/apiRequest.js [hunk 3]
```diff
@@ -4,15 +4,16 @@
...
 const mockUrl = `http://localhost:5000/api/`;
 const localUrl = `http://localhost:8000/api/`;
 
 const getBaseUrl = (params) => {
     if (params.isMocked) return mockUrl;
     if (params.isLocal) return localUrl;
-    return kGlobalConstants.API;
+    // return kGlobalConstants.API;
+    // DEV-5242: Can find good awards via advanced search.
+    return 'https://qat-api.usaspending.gov/api/';
```
Comment (ebdabbs) [inline src/js/helpers/apiRequest.js:16]: Looks like this was a temporary change for testing (not meant to be committed)?

File: src/_scss/pages/award/shared/awardAmountsSection/_charts.scss (hunk 1 above)
Comment (MaxwellKendall) [reply src/_scss/pages/award/shared/awardAmountsSection/_charts.scss:7]: procurement

File: src/js/helpers/apiRequest.js (hunk 3 above)
Comment (MaxwellKendall) [reply src/js/helpers/apiRequest.js:16]: Yes!

**Pull request merged**